packages/babel-plugin-minify-dead-code-elimination/src/remove-use-strict.js

```diff
@@ -0,0 +1,56 @@
+"use strict";
+
+module.exports = removeUseStrict;
+
+const newIssueUrl = "https://github.com/babel/babili/issues/new";
+
+const useStrict = "use strict";
+
+/**
+ * Remove redundant use strict
+ * If the parent has a "use strict" directive, it is not required in
+ * the children
+ *
+ * @param {NodePath} block BlockStatement
+ */
+function removeUseStrict(block) {
+  if (!block.isBlockStatement()) {
```

Comment:
since it is a separate file. to not shoot ourselves in foot by calling `removeUseStrict(SomeOtherPath)`